Documentation/artifact-format.md

```diff
@@ -281,29 +281,30 @@ Format: JSON
 
 ```
 {
-  "updates": [
-    {
-      "type": "rootfs-image"
-    },
-    {
-      "type": "rootfs-image"
+    "updates": [
+        {
+            "type": "rootfs-image"
+        },
+        {
+            "type": "delta-image"
+        }
+    ],
+    "ArtifactProvides": {
+        "artifact_name": "release-2",
+        "artifact_group": "fix",
+        "update_types_supported": [
+            "rootfs-image",
+            "delta-update"
+        ]
     },
-    {
-      ...
+    "ArtifactDepends": {
+        "artifact_name": "release-1",
+        "device_type": [
+            "vexpress-qemu",
+             "beaglebone"
+        ]
     }
-  ],
-  "artifact_provides": [
-          { "artifact_name": "name" },
```

Comment:
It's not necessary for every parameter to be an object is it?

Reply:
Correct me if I'm wrong, but I think that inside a list they have to be.

Reply:
Ah, but you've turned the whole thing from a list to an object... Hmm that might work too, and maybe that's better, since each entry is a key/value pair. Never mind then!